File: JAVA/Program/Shirt.java
class Shirt extends Cloth {
    private String color;
    private String sleeve_type;

    public Shirt(){
        this.color = "";
        this.sleeve_type = "";
    }

    public Shirt(String color, String sleeve_type){
        this.color = color;
        this.sleeve_type = sleeve_type;
    }

    public String get_Color() {
        return color;
    }

    public void set_Color(String color) {
        this.color = color;
    }

    public String get_SleeveType() {
        return sleeve_type;
    }

    public void set_SleeveType(String sleeve_type) {
        this.sleeve_type = sleeve_type;
    }
}

File: PYTHON/Program/Shirt.py
from Cloth import Cloth  

class Shirt(Cloth):
    def __init__(self, color="", sleeve_type=""):
        super().__init__()
        self.color = color
        self.sleeve_type = sleeve_type

    def get_Color(self):
        return self.color

    def set_Color(self, color):
        self.color = color

    def get_SleeveType(self):
        return self.sleeve_type

    def set_SleeveType(self, sleeve_type):
        self.sleeve_type = sleeve_type